recipe/meta.yaml

```diff
@@ -1,43 +1,53 @@
-{% set version = "0.2.3" %}
 {% set name = "intake-parquet" %}
-{% set sha256 = "353078f8b3358abdf0b1fa959fb8bd7f40466c8d04c474f51015b13822b9b8e3" %}
+{% set version = "0.3.0" %}
 
 package:
-   name: {{ name }}
-   version: {{ version }}
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/intake-parquet-{{ version }}.tar.gz
+  sha256: 57e43795f53d5e44d375683f13532bae248005ff428d177613bc7607ce4a0f89
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  skip: true # [py<37 or s390x]
+  script: 
+    # Remove the out of date versioneer.py file from the source for py3.12 issue
+    - rm -f versioneer.py  # [py>311]
```

Comment:
A patch would be better, and maybe submitting a PR to upstream too. Substitute SafeConfigParser for just ConfigParser

Reply:
versioneer.py is a generated file. The best would be for upstream to configure their project to depend on versioneer, instead of using a checked-in version of the generated file. This current solution is acceptable to me.

Reply:
does removal of the file mean that the versions in the pip installed package won't be correctly configured for python 3.12?

Reply:
@danpetry Yeah I am not overly sure about the repercussions of versioneer not being used with 3.12, though with unit tests we can see it is passing the tests (although with an older pyarrow as it hasn't been updated inline it seems)